backend/src/services/website-converter/website-converter-service.ts

```diff
@@ -160,6 +164,52 @@ class WebsiteConverterServiceImpl {
     return this.#filterOnlyBest(tiersWithPokemonDetails);
   }
 
+  public toIngredientRanker(optimalMons: IngredientRankerResult) {
+    const prettifiedCombinations = optimalMons.teams.slice(0, 500).map((solution) => ({
+      team: solution.team
+        .map(
+          (member) =>
+            `${member.pokemonCombination.pokemon.name}(${shortPrettifyIngredientDrop(
+              member.pokemonCombination.ingredientList
+            )})`
+        )
+        .join(', '),
+      details: `👨🏻‍🍳 Team finder - https://sleepapi.net 👨🏻‍🍳\n\nRecipe: ${optimalMons.ingredient} (${
```

Comment:
By default the API returns in proper compact JSON, so this wouldn't be included in the default API anyway unless someone specified pretty output. When you implement the tool on the site, however, this will probably be included in the details section if you're using prettified API. In that case you'll want to update the description here to mention the ingredient tool rather than Team finder. 

Also further down it mentions "requires X mon for this recipe" or something, but this tool always runs with 1 ingredient so the answer would always be 1 mon. Also mentions like "no possible team combinations" which doesn't fit well since we're not finding teams here, but individual mons.

I'm fine keeping this as is for current version though, route isn't used currently and this is only the prettified formatting. Just mentioning it might need another look when the frontend tool is put in place.

Reply:
I made some more edits to the text now anyway. Hopefully I won't need to make many more changes when I actually implement a page for this.